website/docs/api/cli.mdx

```diff
@@ -1296,6 +1296,9 @@ input formats are:
 
 When a directory is provided it is traversed recursively to collect all files.
 
+When loading a .spacy file, any potential annotations stored on the `Doc` will be preserved.
```

Comment:
Mostly for my understanding: whether the annotation will be preserved depends on whether the pipe is set to `overwrite`?

If so, maybe it should say something like:

> When loading a .spacy file, any potential annotations stored on the `Doc` that are not overwritten by a pipe will be preserved.

Reply:
What I intended to say is that annotations are preserved upon loading the `Doc`. It entirely depends on the implementation of the actual pipeline what happens to them afterwards... I'll think about how to rephrase this more clearly.